app/application.js

```diff
@@ -65,7 +67,8 @@ export class DesktopApplication {
   }) {
     this.platform = platform;
     this.isMac = Platforms.isMac(this.platform);
-    app.setName(AppName);
+    app.name = AppName;
+    app.allowRendererProcessReuse = false;
```

Comment:
`false` appears to be the default value for this: https://www.electronjs.org/docs/api/app#appallowrendererprocessreuse

And that this property is going to be removed at some point in the "future".

Reply:
Good catch, should be a `true` there…